examples/css3d_molecules.html

```diff
@@ -334,8 +334,8 @@
 					var geometryBonds = pdb.geometryBonds;
 					var json = pdb.json;
 
-					var offset = geometryAtoms.center();
-					geometryBonds.translate( offset.x, offset.y, offset.z );
+					geometryAtoms.center();
+					geometryBonds.center();
```

Comment:
Does that produce the same offset for both atoms and bonds?

Reply:
The visual result is the same.

BTW: I've missed `webgl_loader_pdb`. Applied the same change there, too.

Reply:
Sorry. How you can center the atoms and bonds separately makes no sense to me.

Reply:
I guess that works because `PDBLoader` builds both geometries from the same vertex data.

Reply:
I would inline the `center()` code, if you have to, and not rely on that.

Reply:
Okay :innocent: